helix-view/src/handlers/lsp.rs

```diff
@@ -72,22 +74,26 @@ impl ToString for ApplyEditErrorKind {
 }
 
 impl Editor {
+    fn uri_to_file_path(&mut self, uri: &helix_lsp::Url) -> Result<PathBuf, ApplyEditErrorKind> {
+        match uri_to_file_path(uri) {
+            Ok(path) => Ok(path),
+            Err(err) => {
+                let err = format!("{err}: {uri}");
+                log::error!("{err}");
+                self.set_error(err);
```

Comment:
I'm not sure we need this error on the statusline - I think the error line in the log is enough

Reply:
agreed although we do statusline report for all the related errors in the file, and did for the `FilePathError::UnableToConvert` case before i refactored it out.